src/core/public/chrome/ui/header/nav_link.tsx

```diff
@@ -63,13 +62,15 @@ export function createEuiListItem({
     href,
     /* Use href and onClick to support "open in new tab" and SPA navigation in the same link */
     onClick(event: React.MouseEvent<HTMLButtonElement, MouseEvent>) {
-      onClick();
+      if (!isModifiedEvent(event)) {
+        onClick();
+      }
+
       if (
         !legacyMode && // ignore when in legacy mode
         !legacy && // ignore links to legacy apps
-        !event.defaultPrevented && // onClick prevented default
```

Comment:
Why was `!event.defaultPrevented` removed?

Reply:
It's not gone, just moved. Merged the check into `[isModifiedEvent()](https://github.com/elastic/kibana/pull/69767/files/43f60ef7dde1566a924cd9a89838c8d2f8c6980f..ddedb8ec358ec4f2791f1f84606af81e8e6aa550#diff-6c559d2add207c8654cc2bb23350384bR27-R28)`. Maybe that's the wrong call if it's confusing but I thought it fit close enough and I needed to do the same check in several (3) places so it seemed convenient.

Reply:
Yea, I missed that in the `isModifiedEvent` diff. Method should probably be renamed to something more explicit. `isModifiedOrPrevented` ?